### PR TITLE
Suggestion for language strings

### PR DESCRIPTION
**Pull request by astridx**
Pull Request as suggestion for Issue #https://github.com/joomla-projects/gsoc18_override_management/issues/58.

templates customise cassiopeia test administration 8

This is just a suggestion because I find the words somewhat confusing.

Especially in the toolbar the word `delete` near `delete file`. It is true that everything is described in the documentary. But the users feel something more user-friendly when it is self-explaining.

But that's all a matter of taste. I know that. I'm not in a huff if this PR will be not merged :)

If you take this words I will update the last picture in 
https://docs.joomla.org/J4.x:Improved_Override_Management

I changed the word `history` in the messages because it only appears here. I think user wonder and asked  "what history?". (This text is not in the doku up to know, so we need not update the doku)

### CODE REVIEW

**Comment by astridx** [comment]
@lane @anu1601cs
What do you think? Should we change the strings or should I close this Pr for tiding up.

**Comment by laoneo** [comment]
Whatever you guys feel ok with.

**Comment by Anu1601CS** [comment]
@astridx @laoneo I am fine with this. Its all about a matter of test. :)

File: administrator/language/en-GB/en-GB.com_templates.ini [hunk 1]
```diff
@@ -170,9 +171,9 @@ COM_TEMPLATES_MSG_MANAGE_NO_STYLES="There are no styles installed matching your
 COM_TEMPLATES_MSG_MANAGE_NO_TEMPLATES="There are no templates installed matching your query."
 COM_TEMPLATES_N_ITEMS_DELETED="%d template styles deleted."
 COM_TEMPLATES_N_ITEMS_DELETED_1="Template style deleted."
-COM_TEMPLATES_N_OVERRIDE_CHECKED="%d override history marked as checked."
-COM_TEMPLATES_N_OVERRIDE_DELEATED="%d Override history deleted."
-COM_TEMPLATES_N_OVERRIDE_UNCHECKED="%d override history marked as unchecked."
+COM_TEMPLATES_N_OVERRIDE_CHECKED="%d Updated files list entry marked as checked."
+COM_TEMPLATES_N_OVERRIDE_DELEATED="%d Updated files list entry deleted."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_templates.ini]: typo
DELETED not DELEATED

**Comment by Anu1601CS** [comment]
@laoneo Should we merge this ?

**Pull request merged**